invent-code-1/main.cpp

```diff
@@ -3,5 +3,10 @@
 using namespace std;
 
 int main() {
-  // Write code here.
+  cout<<"My favorite colour is red. ";
+  cout<<"I was born in Indonesia.";
+  cout<<" I'm 22 years old.";
+  cout<< endl;
 }
+
+publish_all
```

Comment:
I don't think this will compile with the publish_all, you might want to remove it.

Reply:
Done.